contrib/typst-preview/editors/vscode/package.json

```diff
@@ -0,0 +1,289 @@
+            "Scroll preview to current cursor position when selection changes by mouse",
+            "Scroll preview to current cursor position when selection changes by mouse or keyboard (any source)"
+          ]
+        },
+        "typst-preview.partialRendering": {
+          "description": "Only render visible part of the document. This can improve performance but still being experimental.",
+          "type": "boolean",
+          "default": true
+        },
+        "typst-preview.invertColors": {
+          "description": "Invert colors of the preview (useful for dark themes without cost). Please note you could see the origin colors when you hover elements in the preview.",
+          "type": "string",
+          "enum": [
+            "never",
+            "auto",
+            "always"
+          ],
+          "default": "never",
+          "enumDescriptions": [
+            "Disable color inversion of the preview",
+            "Invert colors smartly by detecting dark/light themes in browser environment or by `typst query` your document",
+            "Always invert colors of the preview"
+          ]
+        },
+        "typst-preview.cursorIndicator": {
+          "description": "(Experimental) Show typst cursor indicator in preview.",
+          "type": "boolean",
+          "default": false
+        },
+        "typst-preview.showInActivityBar": {
+          "description": "(Experimental) Show a preview panel in activity bar.",
+          "type": "boolean",
+          "default": false
+        },
+        "typst-preview.statusBarIndicator": {
+          "description": "Style of status bar indicator for typst-preview.",
+          "type": "string",
+          "enum": [
+            "full",
+            "compact"
+          ],
+          "default": "compact",
+          "enumDescriptions": [
+            "Show full status bar indicator (ICON Compile Success)",
+            "Show compact status bar indicator (ICON)"
+          ]
+        },
+        "typst-preview.pinPreviewFile": {
+          "description": "Declare current previewing file as entrypoint for typst-lsp. This will make typst-lsp to use this file as entrypoint instead of the file opened in vscode. This can improve diagnostics messages and auto completion but still being experimental.",
+          "type": "boolean",
+          "default": false
+        }
+      }
+    },
+    "keybindings": [
+      {
+        "command": "typst-preview.preview",
+        "key": "ctrl+k v",
+        "mac": "cmd+k v",
+        "when": "editorLangId == typst"
+      }
+    ]
+  },
+  "scripts": {
+    "build:frontend": "cd ../../../../tools/typst-preview-frontend && yarn run build && cd ../../contrib/typst-preview/editors/vscode && rimraf ./out/frontend/ && cpr ../../../../tools/typst-preview-frontend/dist/ ./out/frontend/ && rimraf ../../../../crates/tinymist-assets/src/typst-preview.html && cpr ./out/frontend/index.html ../../../../crates/tinymist-assets/src/typst-preview.html",
```

Comment:
okay, magic happens here

Reply:
Looks not great, but I just changed the relative paths and doesn't change any other thing.